src/dev/src/app/accessibility/accessibility.demo.routing.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { ModuleWithProviders } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+
+import { AccessibilityDemo } from './accessibility.demo';
+import { AriaLiveServiceDemo } from './aria-live-service.demo';
+
+const ROUTES: Routes = [
+  {
+    path: '',
+    component: AccessibilityDemo,
+    children: [
+      { path: '', redirectTo: 'array', pathMatch: 'full' },
+      { path: 'aria-live', component: AriaLiveServiceDemo },
```

Comment:
Why redirect? Could just navigate to `aria-live`?

Reply:
I had the idea of adding another page, but yes for now is better to skip the redirect